src/hooks/useHasTeam2025Pricing.tsx

```diff
@@ -0,0 +1,21 @@
+import {differenceInDays} from 'date-fns';
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function useHasTeam2025Pricing(): boolean {
+    const [firstPolicyDate] = useOnyx(ONYXKEYS.NVP_PRIVATE_FIRST_POLICY_CREATED_DATE);
+    const [hasManualTeamPricing2025] = useOnyx(ONYXKEYS.NVP_PRIVATE_MANUAL_TEAM_PRICING_2025);
+
+    if (hasManualTeamPricing2025) {
+        return true;
+    }
+
+    if (!firstPolicyDate) {
+        return false;
```

Comment:
Sorry I missed this earlier -

```suggestion
        return true;
```

If they don't have this NVP set, we should offer them the new pricing

Reply:
Done! https://github.com/Expensify/App/pull/59077/commits/1e5e2971c14735caf2c7001c98487d936f577cf1